std_msgs/Int64 grid_path_resolution
nav_msgs/Path[] paths